### DISCUSSION TITLE
カンファレンスの状態の見直し

### DISCUSSION THREAD

**Discussion by takaishi** [cloudnativedaysjp/dreamkast#1368] [Ideas]
2日イベントだと1日目が終わった後にopenedからregisteredに戻すのだけど、そうするとサイトとしては開催前というステータスになるのでアーカイブがみえなかったりする。状態遷移的に微妙なので、開催中だけど1日目のイベントは終了したという状態がほしい。

**Comment by takaishi** [comment]
> 2日イベントだと1日目が終わった後にopenedからregisteredに戻す

@jacopen これってそもそも何のために戻しているのかって覚えてる？

**Comment by takaishi** [comment]
o11y2021では必須ではない（はず）

**Comment by jacopen** [comment]
@takaishi ごめん、コメント貰ってるの見逃してた

これは2つ理由があって
- ダッシュボードを見せて自分の登録セッションを確認できるようにしたい
- 2日目の放送開始前のリハ時に一般参加者が入ってこないようにしたい(Broadcastチームは/ui直叩きで確認する)

という感じです

**Comment by takaishi** [comment]
アーカイブが見える見えないに影響するのでこれどうにかしたい

**Comment by oshiro3** [comment]
直接この話題には関係ないかもですけど、ダッシュボードには運営からのお知らせなども表示するようになったので、障害情報とか随時のお知らせを見てもらう必要もあるのでその辺りも合わせて考えたいかもです

> ダッシュボードを見せて自分の登録セッションを確認できるようにしたい
